src/common/select-objects.js

```diff
@@ -40,7 +40,8 @@ import {
   onChange: propTypes.func,
   placeholder: propTypes.string,
   predicate: propTypes.func,
-  required: propTypes.bool
+  required: propTypes.bool,
+  xoObjects: propTypes.object.isRequired
 })
 class GenericSelect extends Component {
```

Comment:
I'd like a small doc (maybe example) explaining how to use (subclass) GenericSelect.